src/m365/commands/docs.ts

```diff
@@ -0,0 +1,39 @@
+import AnonymousCommand from '../base/AnonymousCommand';
+import { Cli } from '../../cli/Cli';
+import commands from './commands';
+import { Logger } from '../../cli/Logger';
+import type * as open from 'open';
+import { settingsNames } from '../../settingsNames';
+const packageJSON = require('../../../package.json');
+
+class DocsCommand extends AnonymousCommand {
+  private _open: typeof open | undefined;
+
+  public get name(): string {
+    return commands.DOCS;
+  }
+
+  public get description(): string {
+    return 'Returns the CLI for Microsoft 365 docs webpage URL';
+  }
+
+  public async commandAction(logger: Logger): Promise<void> {
+    logger.log(packageJSON.homepage);
+
+    if (Cli.getInstance().getSettingWithDefaultValue<boolean>(settingsNames.autoOpenLinksInBrowser, false) === false) {
+      logger.log(`Use a web browser to open the CLI for Microsoft 365 docs webpage URL`);
+      return;
+    }
+
+    // _open is never set before hitting this line, but this check
+    // is implemented so that we can support lazy loading
+    // but also stub it for testing
+    /* c8 ignore next 3 */
+    if (!this._open) {
+      this._open = require('open');
+    }
+    await (this._open as typeof open)(packageJSON.homepage);
```

Comment:
Can't we write this as?

```suggestion
    await this._open(packageJSON.homepage);
```

Reply:
Casting `this._open` as `typeof open` is necessary because `this._open` may be `undefined`, and attempting to call undefined as a function would result in an error.

I can change it as below with non-null assertion operator, please let me know if it is fine?
`await this._open!(packageJSON.homepage);`

Reply:
Sad that it's not smart enough to see that we are assigning it a value in the condition above. Let's use `!` then indeed.